unattended_installer/builder.sh

```diff
@@ -265,4 +268,26 @@ function builder_main() {
     fi
 }
 
+function checkFilebeatURL() {
+
+    # Import variables
+    . ${resources_installer}/installVariables.sh
+    new_filebeat_url="https://raw.githubusercontent.com/wazuh/wazuh/master/extensions/elasticsearch/7.x/wazuh-template.json"
+
+    # Get the response of the URL and check it
+    response=$(curl --write-out '%{http_code}' --silent --output /dev/null $filebeat_wazuh_template)
+    if [ $response != "200" ]; then
+       	response=$(curl --write-out '%{http_code}' --silent --output /dev/null $new_filebeat_url)
+
+        # Display error if both URLs do not get the resource
+        if [ $response != "200" ]; then
+            echo -e "Error: Could not get the Filebeat Wazuh template. "
+        # If matches, replace the variable of installVariables to the new one
+        else
+            echo -e "Changing Filebeat URL..."
+            sed -i -E "s/http.+\\$\\{wazuh_major\\}.+wazuh-template.json/https:\/\/raw.githubusercontent.com\/wazuh\/wazuh\/master\/extensions\/elasticsearch\/7.x\/wazuh-template.json/" ${resources_installer}/installVariables.sh
```

Comment:
Instead of changing the `filebeat_wazuh_template` entire value, it could be easier to change the minor variable value by `master`, right?